WindowsServerDocs/administration/server-core/what-is-server-core.md

```diff
@@ -37,7 +37,25 @@ In addition to no UI, Server Core also differs from the Server with Desktop Expe
 - No OOBE (out-of-box-experience) for setting up Server Core
 - No audio support
 
-The following table shows which applications are available *locally* on Server Core vs Server with Desktop Experience. **Important**: In most cases, applications that are listed as "not available" below can be run remotely from a Windows client computer and used to manage your Server Core installation.
+The following table shows which applications are available *locally* on Server Core vs Server with Desktop Experience. **Important**: In most cases, applications that are listed as "not available" below can be run remotely from a Windows client computer or [Windows Admin Center](../../manage/windows-admin-center/overview.md) to manage your Server Core installation. 
+
+### Versioning and Downloads of Feature on Demand for Server Core
+
+The list below reflects the FoD level as of Version 1809 (Server 2019 LTSC). 
+You can greatly enchance your Server Core compatibility by applying a FoD package, newer than 1809.
+
+*Download locations*
```

Comment:
This line looks like it should have been a subheading for the previous H3 heading, so perhaps use a H4 heading here?
```suggestion
#### Download locations
```